modules/registry-refactoring/src/main/resources/appcatalog-mysql.sql

```diff
@@ -568,6 +568,85 @@ CREATE TABLE USER_COMPUTE_RESOURCE_PREFERENCE (
   PRIMARY KEY (RESOURCE_ID,USER_ID,GATEWAY_ID)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+CREATE TABLE GROUP_RESOURCE_PROFILE (
+  GATEWAY_ID varchar(255) NOT NULL,
+  GROUP_RESOURCE_PROFILE_ID varchar(255) NOT NULL,
+  GROUP_RESOURCE_PROFILE_NAME varchar(255) DEFAULT NULL,
+  CREATION_TIME BIGINT NOT NULL,
+  UPDATE_TIME BIGINT NOT NULL,
+  PRIMARY KEY (GATEWAY_ID,GROUP_RESOURCE_PROFILE_ID)
+)ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+CREATE TABLE BATCH_QUEUE_RESOURCE_POLICY (
+  RESOURCE_POLICY_ID varchar(255) NOT NULL,
+  COMPUTE_RESOURCE_ID varchar(255) NOT NULL,
+  GROUP_RESOURCE_PROFILE_ID varchar(255) NOT NULL,
+  GATEWAY_ID varchar(255) NOT NULL,
+  QUEUE_NAME varchar(255) NOT NULL,
+  MAX_ALLOWED_NODES INTEGER,
+  MAX_ALLOWED_CORES INTEGER,
+  MAX_ALLOWED_WALLTIME INTEGER,
+  PRIMARY KEY (RESOURCE_POLICY_ID,COMPUTE_RESOURCE_ID,GROUP_RESOURCE_PROFILE_ID,GATEWAY_ID,QUEUE_NAME),
+  FOREIGN KEY (COMPUTE_RESOURCE_ID,QUEUE_NAME) REFERENCES BATCH_QUEUE(COMPUTE_RESOURCE_ID,QUEUE_NAME) ON DELETE CASCADE,
+  FOREIGN KEY (GATEWAY_ID,GROUP_RESOURCE_PROFILE_ID) REFERENCES GROUP_RESOURCE_PROFILE(GATEWAY_ID,GROUP_RESOURCE_PROFILE_ID) ON DELETE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+CREATE TABLE COMPUTE_RESOURCE_POLICY (
+    RESOURCE_POLICY_ID varchar(255) NOT NULL,
+    COMPUTE_RESOURCE_ID varchar(255) NOT NULL,
+    GROUP_RESOURCE_PROFILE_ID varchar(255) NOT NULL,
+    GATEWAY_ID varchar(255) NOT NULL,
+    PRIMARY KEY (GATEWAY_ID,RESOURCE_POLICY_ID,COMPUTE_RESOURCE_ID,GROUP_RESOURCE_PROFILE_ID),
+    FOREIGN KEY (COMPUTE_RESOURCE_ID) REFERENCES COMPUTE_RESOURCE(RESOURCE_ID) ON DELETE CASCADE,
+    FOREIGN KEY (GATEWAY_ID,GROUP_RESOURCE_PROFILE_ID) REFERENCES GROUP_RESOURCE_PROFILE(GATEWAY_ID,GROUP_RESOURCE_PROFILE_ID) ON DELETE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+CREATE TABLE COMPUTE_RESOURCE_POLICY_QUEUES (
```

Comment:
Let's make a FK to BATCH_QUEUE here. And the PK can be (RESOURCE_POLICY_ID, QUEUE_NAME).